mod-notify/mod-notify.postman_collection.json

```diff
@@ -4807,29 +4651,29 @@
 							"listen": "test",
 							"script": {
 								"id": "926890c6-7d6e-4060-b6bf-c52ff73b5974",
-								"type": "text/javascript",
 								"exec": [
 									"pm.test(\"Status code is 404\", function () {",
 									"    pm.response.to.have.status(404);",
 									"});   ",
 									"    ",
 									"pm.test(\"Response contains correct data\", function () { ",
-									" pm.expect(pm.response.text()).to.equal(\"No suitable module found for path /notify/_username\");",
+									" pm.expect(pm.response.text()).to.equal(\"No suitable module found for path /notify/_username for tenant fs00000000\");",
```

Comment:
Can we use the env/global variable to look up the tenant dynamically? It's likely always this one and that the message may be changed in the future, but if we do not hard code it then it'd be better.

Reply:
Just seeing Kruthi's comment now. I'll delete this one :)